src/com/google/enterprise/adaptor/database/DatabaseAdaptor.java

```diff
@@ -678,6 +678,9 @@ public void getDocContent(Request req, Response resp) throws IOException {
       // Generate Acl if aclSql is provided.
       if (aclSql != null) {
         resp.setAcl(getAcl(conn, id.getUniqueId()));
+      } else {
+        resp.setAcl(buildAcl(rs, aclPrincipalDelimiter, aclNamespace,
```

Comment:
add comment 
// see if ACL came as part of result to singleDocContentSql query

Reply:
Done